alicloud/resource_alicloud_hbr_nas_backup_plan.go

```diff
@@ -0,0 +1,513 @@
+package alicloud
+
+import (
+	"fmt"
+	"log"
+	"strconv"
+	"time"
+
+	util "github.com/alibabacloud-go/tea-utils/service"
+	"github.com/aliyun/terraform-provider-alicloud/alicloud/connectivity"
+	"github.com/hashicorp/terraform-plugin-sdk/helper/resource"
+	"github.com/hashicorp/terraform-plugin-sdk/helper/schema"
+	"github.com/hashicorp/terraform-plugin-sdk/helper/validation"
+)
+
+func resourceAlicloudHbrNasBackupPlan() *schema.Resource {
+	return &schema.Resource{
+		Create: resourceAlicloudHbrNasBackupPlanCreate,
+		Read:   resourceAlicloudHbrNasBackupPlanRead,
+		Update: resourceAlicloudHbrNasBackupPlanUpdate,
+		Delete: resourceAlicloudHbrNasBackupPlanDelete,
+		Importer: &schema.ResourceImporter{
+			State: schema.ImportStatePassthrough,
+		},
+		Schema: map[string]*schema.Schema{
+			"backup_type": {
+				Type:         schema.TypeString,
+				Optional:     true,
+				Computed:     true,
+				ForceNew:     true,
+				ValidateFunc: validation.StringInSlice([]string{"COMPLETE"}, false),
+			},
+			"bucket": {
+				Type:     schema.TypeString,
+				Optional: true,
+			},
+			"create_time": {
+				Type:     schema.TypeString,
+				Required: true,
+				ForceNew: true,
+			},
+			"detail": {
+				Type:     schema.TypeString,
+				Optional: true,
+			},
+			"disk_id": {
+				Type:     schema.TypeString,
+				Optional: true,
+			},
+			"exclude": {
+				Type:     schema.TypeString,
+				Optional: true,
+			},
+			"file_system_id": {
+				Type:     schema.TypeString,
+				Optional: true,
+				ForceNew: true,
+			},
+			"include": {
+				Type:     schema.TypeString,
+				Optional: true,
+			},
+			"instance_id": {
+				Type:     schema.TypeString,
```

Comment:
I think the instance_id and disk_id are redundant in the nas_backup_plan resource.